src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -244,7 +244,6 @@ kibana_vars=(
     xpack.apm.serviceMapFingerprintGlobalBucketSize
     xpack.apm.ui.enabled
     xpack.apm.ui.maxTraceItems
-    xpack.apm.ui.transactionGroupBucketSize
```

Comment:
This seems like a breaking change - I skipped the comments but may have missed the context, was there previous discussion?

I'm less concerned about the behavior change and more that previous kibana.yml's with this config won't start anymore.

Reply:
I think it's fine to remove that setting (it's no longer used) but we should mark it as such in `apm/server/index.ts` similar to what we did for `indices.sourcemap`:

https://github.com/elastic/kibana/blob/16d69653a0361d12a10a5d8be15d64eee8a12548/x-pack/plugins/apm/server/index.ts#L68

Reply:
Can you clarify the behavior of:

8.6 `kibana.yml` with `xpack.apm.ui.transactionGroupBucketSize` defined, and then upgraded to 8.7?  Will Kibana start?

Reply:
If we add the `unused` above it'll start with s warning, yes.
We can also reduce the log level to info btw

Reply:
the log level could be only `critical` or `warning`.
<img width="602" alt="image" src="https://user-images.githubusercontent.com/1313018/212297903-da265292-5804-401d-a7a4-511aca5eb3ef.png">

> If the setting is mark as unused Kibana should start with a warning (@yngrdyn please double check this is the case).

To validate this, I added `xpack.apm.ui.transactionGroupBucketSize: 1000` in `kibana.yml` file after removing the option from the source code. Kibana was restarted successfully and got this warning in the logs

<img width="1083" alt="image" src="https://user-images.githubusercontent.com/1313018/212301052-94627cd6-b14b-4d38-b048-e7e479dd5759.png">

Reply:
Thanks for checking @yngrdyn. Let's leave it as warning.

Reply:
Thanks for the clarification.  Thoughts on leaving the docker line in?  If it doesn't exist the setting will never be passed to kibana in container + environment variable configuration - may cause some confusion.

Reply:
I'm fine with either. Do you suggest leaving it in?

Reply:
@jbudz are you suggesting leaving the setting in the file or removing it?

Reply:
Yes, leaving it in.